.github/actions/setup-pip/action.yml

```diff
@@ -21,10 +24,12 @@ runs:
     - name: Install pip
       shell: bash
       run: |
-        pip install --no-cache-dir --upgrade "pip>=20.0.2"
+        make upgrade-pip
```

Comment:
This is the key to making the CI and development use the same "pip upgrade" path.